yourgov/components/FormMobileFoodVendorPermit/FormActions.tsx

```diff
@@ -0,0 +1,56 @@
+import { Fragment } from 'react';
+import { Button, ButtonGroup } from '@ag.ds-next/react/button';
+import { Modal } from '@ag.ds-next/react/modal';
+import { Stack } from '@ag.ds-next/react/stack';
+import { Text } from '@ag.ds-next/react/text';
+import { useTernaryState } from '@ag.ds-next/react/core';
+import { Divider } from '@ag.ds-next/react/divider';
+import { useGlobalForm } from './GlobalFormProvider';
+
+export function FormActions() {
+	const [isModalOpen, openModal, closeModal] = useTernaryState(false);
+
+	const { isSubmittingStep, saveAndExit, isSavingBeforeExiting, cancel } =
+		useGlobalForm();
+
+	return (
+		<Fragment>
+			<Stack gap={3}>
+				<Divider />
+				<ButtonGroup>
+					<Button type="submit" variant="primary" loading={isSubmittingStep}>
+						Save and continue
+					</Button>
+					<Button
+						type="button"
+						variant="secondary"
+						loading={isSavingBeforeExiting}
+						onClick={saveAndExit}
+					>
+						Save and exit
+					</Button>
+					<Button type="button" variant="tertiary" onClick={openModal}>
```

Comment:
This feels like a weird button to be on the final review and submit page especially. I did not expect it to destroy all of my entered data

Reply:
Did you get the modal?

<img width="743" alt="Screenshot 2023-12-20 at 9 49 01 am" src="https://github.com/agriculturegovau/agds-next/assets/6265154/90144609-5fb4-4661-b222-9263b5096df5">

Reply:
I did, but I still didn't expect it to be quite as destructive as it here, since I didn't enter any data on this page. I dunno, but it feels like Cancel should only delete the data on the particular step, and to delete all should be done from the table list.